pootle/apps/accounts/utils.py

```diff
@@ -30,3 +30,38 @@ def get_user_by_email(email):
             return User.objects.get(email__iexact=email)
         except User.DoesNotExist:
             return None
+
+
+def merge_user(src, target):
```

Comment:
Please use `src_user` and `target_user` instead. They are more verbose and less prone to produce confussion.